bundles/org.eclipse.jface/src/org/eclipse/jface/resource/JFaceResources.java

```diff
@@ -597,8 +597,9 @@ public static void setFontRegistry(FontRegistry registry) {
 	 *
 	 * @param owner control whose disposal will trigger cleanup of everything in the
 	 *              registry.
+	 * @return LocalResourceManager
 	 */
-	static LocalResourceManager managerFor(Control owner) {
+	static public LocalResourceManager managerFor(Control owner) {
```

Comment:
The usual convention is to mention the visibility first.
```suggestion
	public static LocalResourceManager managerFor(Control owner) {
```